app/models/price_group.rb

```diff
@@ -20,9 +20,17 @@ class PriceGroup < ActiveRecord::Base
   scope :for_facility, -> (facility) { where(facility_id: [nil, facility.id]) }
   scope :globals, -> { where(facility_id: nil) }
 
-  scope :base, -> { globals.where(name: Settings.price_group.name.base) }
-  scope :external, -> { globals.where(name: Settings.price_group.name.external) }
-  scope :cancer_center, -> { globals.where(name: Settings.price_group.name.cancer_center) }
+  def self.base
```

Comment:
We'd see test failures on merge PRs, no? Is that something that makes sense to handle as part of a merge, since those changes would need to happen on the individual forks?